Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1169,7 +1169,7 @@ void GridFormattingContext::expand_flexible_tracks(AvailableSpace const& availab
     auto& tracks = dimension == GridDimension::Column ? m_grid_columns : m_grid_rows;
     auto& available_size = dimension == GridDimension::Column ? available_space.width : available_space.height;
 
-    auto find_the_size_of_an_fr = [&](Vector<GridTrack&> tracks, CSSPixels space_to_fill) -> CSSPixels {
+    auto find_the_size_of_an_fr = [&](Vector<GridTrack&> const& /*FIXME: Span*/ tracks, CSSPixels space_to_fill) -> CSSPixelFraction {
```

Comment:
hmm, do we really want to use `CSSPixelFraction` for `fr` size? It should be ok to store it using `double` until it is applied to resolve width/height of a track.

Reply:
At some point we'll have a `Flex` class for these. I just haven't got around to it yet. 😅

Reply:
Well previously it was a `CSSPixels`, I just noticed that it may return a fraction and will be scaled later (by a double) so making it a fraction adds some precision